content/en/PentestStates/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Pentest States"
+linkTitle: "Pentest States"
+weight: 500
+description: >
+  Status of your pentest.
+---
+
+{{% pageinfo %}}
+Learn the meaning of the labels associated with your pentests.
+{{% /pageinfo %}}
+
+From the moment you've started saving a draft of a pentest, you'll see a label
+associated with that pentest in various locations. We've set up these definitions
+as if you've set up the pentest through our user interface. Here's the meaning of each
+label:
+
+| Label       | Description                                                                                                                                                                                                                               |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Draft       | You’ve started the **Create a Pentest** process. However, you may not have added the information that we need to start work on your pentest.                                                                                              |
+| In Review   | You’ve submitted the pentest. We’re reviewing your submission. We may have questions for you as we review your submission.                                                                                                                |
+| Planned     | We’ve accepted your proposed pentest, and have assigned pentesters. You should also have access to a Slack channel dedicated to your pentest.                                                                                             |
+| Live        | We’ve started working on your pentest. Watch your Slack channel. When we have questions and discover findings, we’ll share them in the channel.                                                                                           |
```

Comment:
should we specify 'Slack channel'?

Reply:
You're right, I'm changing to something more generic.

```suggestion
| Live        | We’ve started working on your pentest. Watch the pentest chat channel. When we have questions and discover findings, we’ll share them in the channel.                                                                                           |
```

Reply:
The "Commit suggestion" button isn't working. I'll make the change "the old fashioned way". :)